content/operate/rc/databases/connect.md

```diff
@@ -55,10 +55,12 @@ A Redis client is a software library or tool that enables applications to intera
 
 The connection wizard provides code snippets to connect to your database with the following programming languages:
 
-- node.js using [node-redis]({{< relref "/develop/clients/nodejs" >}})
 - .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
```

Comment:
You probably *do* only need the third-party StackExchange.Redis stuff to connect, but I'd say it's better to point users to our docs (which also cover connection, etc).
```suggestion
- .NET using [NRedisStack]({{< relref "/develop/clients/dotnet" >}})
```